app/public/cantusdata/views/mei_submission.py

```
@@ -0,0 +1,72 @@
+    submitting application, not to the person who did the OMR work -- the
+    submitter's own username travels in the request body. Permissions are the
+    model's: POST needs `cantusdata.add_meisubmission`, GET needs
+    `cantusdata.view_meisubmission`, both grantable from the admin.
+
+    Note that these are set explicitly because the project configures no
+    DEFAULT_PERMISSION_CLASSES, so DRF's default is AllowAny; naming
+    TokenAuthentication explicitly also keeps CSRF out of a server-to-server POST.
+    """
+
+    queryset = MEISubmission.objects.all()
+    authentication_classes = [TokenAuthentication]
```

Ruff (0.16.2)

[warning] Mutable default value for class attribute

(RUF012)